SofaKernel/modules/SofaBaseMechanics/SubsetMapping.inl

```diff
@@ -187,7 +191,15 @@ template <class TIn, class TOut>
 void SubsetMapping<TIn, TOut>::apply ( const core::MechanicalParams* /*mparams*/, OutDataVecCoord& dOut, const InDataVecCoord& dIn )
 {
     const IndexArray& indices = f_indices.getValue();
-
+    
+    if (f_resizeToModel.getValue() || this->toModel->getSize() < indices.size())
+    { 
+        if (this->toModel->getSize() != indices.size()) 
```

Comment:
There are existing scenes that rely on SubsetMapping not resizing to the size of the indices array, when for instance creating 20 points but only the first 10 are updated by the mapping.

The design of most changes in Sofa in our branches is to never break existing run-time behavior of such commonly-used component, new behaviors (such as resizing here, or the disabling of the topo engine) are opt-in through an added flag).

If there is a decision regarding the behavior of standard mappings to resize or not the output model, then existing mapping should be updated but with a way to warn developers if they are potentially affected by the change of default behavior.